### ISSUE TITLE
Inputs from coaching session

### ISSUE THREAD

**Issue by mauricebasement** [mauricebasement/GardenControl#6]
We need to check first what has been done already: Is it possible to contribute to an existing project? Our application is basically the most popular amongst arduino apllications (Benefit of an already exisitng community). Maybe there is something developed as far that the challange is only marketing it. Or a project can be used as a framework to implement the advanced GUI ideas. 

As well I have to redefine my role in the project. I have to do proper time management with all my projects, which means that I can not activley support development of software, as it involves learning a lot of skills. I can finacially support software development by crowdsourcing, help to make decisions, and identifying tasks. It is a question as well how deep you wanna get into it, or if we should use crowd funding mostley. I rather want to use skills I already have, so once we get to the hardware stage, I want to manage getting the project physical. We have the proof of concept of electronics, so it makes no sense to mix this two stages more than necessary for testing the software.

**Comment by jgtaylor** [comment]
Agreed - it makes sense to re-use what is available. Unfortunately, there are very few hardware agnostic projects. As far as I know, only cylon.js is actively attempting to support multiple platforms. One of my primary goals is the framework that enables multi-platform access.

This means that hardware can and _should_ go forward; there is no reason why hardware designs should be limited by (current) software. This means, from my perspective, that any modules can be used, from a single Wemos D1 with a single temp sensor, to a network of devices with sensors and actuators. The idea behind the framework is 'interoperability' - that is, any connected module should be able to watch and respond to any other connected module.

As for the roles and time - you have to make sure all your bases are covered; to that end, I wouldn't expect you to work on this full-time. If you're able to get hardware produced (via crowd sourcing, or any other means), I'm all for it! I would be happy to put in feature requests :-) I, on the other hand, have lots of time and no money, so I don't mind the Software dev aspect. Though, I imagine once the framework is 'well defined', sourcing that wouldn't be too difficult, but without a good definition, it will just be useless costs, until we iterate a final architecture.
